OpenSim/Common/Exception.h

```diff
@@ -276,7 +276,9 @@ class KeyNotFound : public Exception {
 
 class IOError : public Exception {
 public:
+#ifndef SWIG
```

Comment:
According to swig [docs](http://www.swig.org/Doc3.0/CPlusPlus11.html#CPlusPlus11_object_construction_improvement) this shouldn't be necessary. SWIG should at least parse correctly.

Reply:
Ah I was able to remove the changes to this file. Consistent with the documentation, there is no warning on this line.

However, there is a warning in `TRCFileAdapter.h:32`:

```
/Users/chris/repos/opensim/13/opensim-core/OpenSim/Common/TRCFileAdapter.h:32: Wa
rning 315: Nothing known about 'IOError::IOError'.  
```